### PR TITLE
Enable authors to delete events

### PR DESCRIPTION
**Pull request by mauricioabreu**

### CODE REVIEW

**Comment by mauricioabreu** [comment]
It solves #215

**Comment by luanfonceca** [comment]
You should run the `makemessages` to create the new .po files with your new strings like: `Event deleted`.

File: deck/templates/event/event_confirm_delete.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <button type="submit" class="btn-flat success text-upper">
+          <i class="icon icon-trash"></i>
+          {% trans "Delete" %}
+        </button>
+        <div class="pull-right">
+          <a href="{% url 'view_event' slug=object.slug %}" class="link text-upper">
+            <i class="icon-calendar"></i>
+            {% trans "event detail" %}
+          </a> |
+          <a href="{% url 'my_proposals' %}" class="link text-upper">
```
Comment (luanfonceca) [inline deck/templates/event/event_confirm_delete.html]: I dont think that this link should be in here

**Comment by mauricioabreu** [comment]
@luanfonceca Fixed

**Comment by luanfonceca** [comment]
Thanks @mauricioabreu 


I was missing you on the contributions <3

**Comment by mauricioabreu** [comment]
@luanfonceca hahaha <3

envergonhado

**Comment by mauricioabreu** [comment]
@luanfonceca is Travis disabled?

**Comment by luanfonceca** [comment]
@mauricioabreu you create the translation files but didint translate it right? Because there are a few fuzzy headers on the portuguese translation.

**Comment by mauricioabreu** [comment]
I forgot to translate. I am gonna change it asap.

**Comment by mauricioabreu** [comment]
You already did. THANK YOU! <3

**Comment by luanfonceca** [comment]
<3

**Comment by luanfonceca** [comment]
I just deploy it and everything is running fine o/

**Pull request merged**